params/config.go

```diff
@@ -778,9 +695,9 @@ func (c *ChainConfig) rules(num *big.Int, timestamp uint64) Rules {
 	}
 }
 
-// AvalancheRules returns the Avalanche modified rules to support Avalanche
+// Rules returns the Avalanche modified rules to support Avalanche
 // network upgrades
-func (c *ChainConfig) AvalancheRules(blockNum *big.Int, timestamp uint64) Rules {
+func (c *ChainConfig) Rules(blockNum *big.Int, timestamp uint64) Rules {
```

Comment:
if we rename this method to `Rules`, like it is in upstream, should we also take the same bool `isMerge` parameter?
Honestly it's kinda weird that they have the isMerge parameter, but I think if the name matches the signature should also match, otherwise perhaps it's better to leave it as AvalancheRules.

Reply:
IMO we should not use that `isMerge` param. but I also slightly think it is probably better to use same `Rules` function. I can revert that if you think we should be using `AvalancheRules` instead.